src/features/myProtectedData/MyProtectedData.tsx

```diff
@@ -118,7 +118,7 @@ export default function MyProtectedData() {
                     <div className="max-w-[300px] flex-1">
                       <ProtectedDataCard
                         id={address}
-                        title={name || 'Undefined'}
+                        title={name || '(No name)'}
```

Comment:
It made me think there was a display bug when seeing this `undefined` value, so I'm changing for something more explicit.

<img width="880" alt="Capture d’écran 2023-12-27 à 15 34 02" src="https://github.com/iExecBlockchainComputing/web3mail-usecase-demo/assets/4280765/30fb0e88-7551-4044-b72c-4c79ffbc8901">